Example/Glia Sample Integration/Views/ContentView.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct ContentView: View {
+
+    @ObservedObject var presenter = ContentViewPresenter()
+
+    var body: some View {
+        VStack(spacing: 16){
```

Comment:
I see here and in the EngagementButtonStyle that the spacing, paddings, etc are used directly in the code. Is that how normally they are made in SwiftUI? Because normally I would advise to move them to a variable. If you want to create another VStack inside in the future, does that get the parent's spacing or you would have to set it again? In that case, it would be better to add it to a variable.

Reply:
Short answer: Yes in SwiftUI they are used directly in the code.
A bit longer. I use those directly because it is a static modifier. If for instance there was a condition for having padding or spacings dynamic, then most definitely they will be stored in a variable. So in the current situation `let spacing = 16` would be an overkill.

To answer your question about Vstack: VStack within a VStack won't get the spacing of the parent but is rather considered as a single entity(view) within the parent VStack. This would mean that the child VStack will have its own spacing if declared so. This is useful because if your spacing within parent and child is the same, there would be no need for the child in the first place.